### PR TITLE
Offline

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

File: app/_posts/2014-12-03-offline.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Offline
+
+What is an offline web application? How can you download when you're offline? Web pages are things you download and render. Downloading implies a network connection. Of course you can't download when you're offline, but you can download when you're online and that's how HTML5 web applications work.
+
+## appcache
```
Comment (simonwex) [inline app/_posts/2014-12-03-offline.md]: Need to explain that the user does have to be initially connected for this to work.

File: app/_posts/2014-12-03-offline.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Offline
+
+What is an offline web application? How can you download when you're offline? Web pages are things you download and render. Downloading implies a network connection. Of course you can't download when you're offline, but you can download when you're online and that's how HTML5 web applications work.
```
Comment (simonwex) [inline app/_posts/2014-12-03-offline.md]: This is a bit basic for our audience.

File: app/_posts/2014-12-03-offline.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+#### Enabling the application cache
+
+```
+<html manifest="example.appcache">
+  ...
+</html>
+```
+
+
+### WIP
```
Comment (simonwex) [inline app/_posts/2014-12-03-offline.md]: I'd like to see more about:

 - how we should save user state
 - what to do when we need a connection for a user action

**Comment by alicoding** [comment]
@simonwex  I have added some more information about user's state.

**Pull request merged**